docs/terminal/basics.md

```diff
@@ -22,6 +22,23 @@ To open the terminal:
 
 > **Note:** Open an external terminal with the `kb(workbench.action.terminal.openNativeConsole)` keyboard shortcut if you prefer to work outside VS Code.
 
+## Opening terminals from the Explorer
+
+You can open new terminals for specific folders from the Explorer via the **Open in Integrated Terminal** context menu command.
+
+![Open in Integrated Terminal command in the Explorer context menu](images/basics/open-in-terminal-command.png)
```

Comment:
This isn't great for screen readers which would read something like below, just repeating itself:

> You can open new terminals for specific folders from the Explorer via the **Open in Integrated Terminal** context menu command. Image: Open in Integrated Terminal command in the Explorer context menu

Reply:
Alt text like "A folder selected in the Explorer displaying the context menu with the Open in Integrated Terminal command"